cmd/gradleExecuteBuild.go

```diff
@@ -67,14 +71,20 @@ initscript {
     }
   }
   dependencies {
-    classpath "com.cyclonedx:cyclonedx-gradle-plugin:1.5.0"
+    classpath "org.cyclonedx:cyclonedx-gradle-plugin:1.7.0"
```

Comment:
should be pin the version for maven as well?

Reply:
Yes, this should be done for all integrations to avoid unplanned breaking of downstream processes